Makefile

```diff
@@ -4,7 +4,7 @@ test: test_no_proxy test_proxy
 
 test_no_proxy:
 	@echo "Running all tests with no proxy process"
-	CODEJAIL_PROXY=0 nosetests
+	CODEJAIL_PROXY=0 nosetests --with-xunit --xunit-file reports/nosetests-proxy.xml
```

Comment:
I'm not really familiar with this repo at all, but do we also want to add this option to the `test_proxy` make command? And do we also want to call this `nosetests-no-proxy` or something?

Reply:
Ah good catch on the name. Fixing. As for running both, I mentioned that in the PR body.  We are essentially running the tests multiple times in different ways, and it would overwrite/munge the xunit.xml.